src/extra/treemacs-all-the-icons.el

```diff
@@ -0,0 +1,251 @@
+;;; treemacs-all-the-icons.el --- all-the-icons integration for treemacs -*- lexical-binding: t -*-
+
+;; Copyright (C) 2020 Eric Dallo
```

Comment:
All right, one final thing. For technical correctness I think this part needs to go back to my name - it'll be merged in my package and I'll be responsible for maintaining it in the future.

Reply:
Hold on, you reverted too far. I only meant the copyright, you should still be the author.

Reply:
Oh, sorry, I see

Reply:
Done!